tests/unit/services/unified-event-handler-test.js

```diff
@@ -227,9 +232,11 @@ test('triggerEvent triggers the event at a throttled rate', function(assert) {
   let throttleSpy = sandbox.spy(Ember.run, 'throttle');
```

Comment:
do we care if throttle is called? This feels needless coupling. Ultimately, all we care is that the callbacks1 and 2 are called appropriately.  We should test that, and only that.

Reply:
Thought about this for a while. While I think the fact that methods are throttled is important, there is no logic wrapping the throttling functionality, so probably it's not necessary to unit tests. Will remove.